x-pack/plugins/observability_solution/slo/public/hooks/use_clone_slo.ts

```diff
@@ -11,22 +11,24 @@ import { SLOWithSummaryResponse } from '@kbn/slo-schema';
 import { useKibana } from '../utils/kibana_react';
 import { paths } from '../../common/locators/paths';
 
-export function useCloneSlo() {
+export function useCloneSlo(remoteKibanaUrl?: string) {
   const {
     http: { basePath },
     application: { navigateToUrl },
   } = useKibana().services;
 
   return useCallback(
     (slo: SLOWithSummaryResponse) => {
-      navigateToUrl(
-        basePath.prepend(
-          paths.sloCreateWithEncodedForm(
-            encode({ ...slo, name: `[Copy] ${slo.name}`, id: undefined })
-          )
-        )
+      const clonePath = paths.sloCreateWithEncodedForm(
+        encode({ ...slo, name: `[Copy] ${slo.name}`, id: undefined })
       );
+      navigateToUrl(basePath.prepend(clonePath));
+      if (remoteKibanaUrl) {
+        window.open((slo.kibanaUrl + clonePath).replace(/\/\//g, '/'), '_blank');
```

Comment:
I see why we have the `remoteName` query parameter used into the remote kibana URL. I guess we can live with that.